proto/penumbra/penumbra/core/component/shielded_pool/v1/shielded_pool.proto

```diff
@@ -10,7 +10,7 @@ import "penumbra/crypto/tct/v1/tct.proto";
 
 // Configuration data for the shielded pool component.
 message ShieldedPoolParameters {
-  FmdParameters fixed_fmd_params = 1;
+  FmdMetaParameters fmd_meta_params = 1;
```

Comment:
I don't think we want to do this, the intention behind calling it `fixed_fmd_parameters` was to clarify that it was a placeholder.

Rather than aiming at serialization compatibility of the new meta-parameters, we should do
```
FmdMetaParameters fmd_meta_params = 2;
```
and mark the existing field as deprecated. Then the new struct doesn't need to be constrained by the existing data.

(As is, the change is a breaking protobuf change, we're not allowed to change field names).